src/storybook/components/information-box/information-box.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import { BEMClass } from "../../../helpers/bem-helper";
+import "./information-box.scss";
+
+const BASE_CSS_CLASS = "monday-storybook-information-box";
+const bemHelper = BEMClass(BASE_CSS_CLASS);
+
+export const InformationBox = ({ component, title, description }) => {
+  return (
```

Comment:
the related components structure repeat itself multiple time in the welcome page (title with the same appearance, description with the same appearance) so I create InformationBox that from now on related component use this